.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: CI Check
```

Comment:
nit: should this be named in way to reflect that this is a GitHub actions build?